### PR TITLE
lndclient: add subscribe channel backups to client

### PR DESCRIPTION
**Pull request by blackjid**
This would help users to build solutions to work with channel backup snapshots.

It adds a function to `lndclient` to subscribe to new channel backups.

### CODE REVIEW

File: lightning_client.go [hunk 1]
```diff
@@ -84,6 +84,12 @@ type LightningClient interface {
 	// chanbackup.Multi payload.
 	ChannelBackups(ctx context.Context) ([]byte, error)
 
+	// SubscribeChannelBackups allows a client to sub-subscribe to the
+	// most up to date information concerning the state of all channel
+	// backups
```
Comment (guggero) [inline lightning_client.go:89]: nit: punctuation missing.

File: lightning_client.go [hunk 2]
```diff
@@ -1436,6 +1442,42 @@ func (s *lightningClient) ChannelBackups(ctx context.Context) ([]byte, error) {
 	return resp.MultiChanBackup.MultiChanBackup, nil
 }
 
+func (s *lightningClient) SubscribeChannelBackups(ctx context.Context) (
```
Comment (guggero) [inline lightning_client.go:1445]: nit: Should have a godoc comment as well. Can just copy from the interface above.

**Review by guggero — COMMENTED** [review]
Thanks for the PR!
Just two small nits, otherwise looks good to me :tada:

**Review by guggero — APPROVED** [review]
LGTM, thanks :tada:

File: lightning_client.go [hunk 3]
```diff
@@ -1436,6 +1442,45 @@ func (s *lightningClient) ChannelBackups(ctx context.Context) ([]byte, error) {
 	return resp.MultiChanBackup.MultiChanBackup, nil
 }
 
+// SubscribeChannelBackups allows a client to sub-subscribe to the
```
Comment (carlaKC) [inline lightning_client.go:1445]: nit: `sub-subscribe` -> `subscribe`?

**Review by carlaKC — APPROVED** [review]
Thanks for the PR! Just a nit on one of the comments from me, non-blocking 🌮

File: lightning_client.go (hunk 3 above)
Comment (blackjid) [reply lightning_client.go]: ups it doesn't make sense.. I thing...... I copy/paste that part from `lnd/lnrpc/rpc.proto` 🤷‍♂️

**Pull request merged**